test/distribution/test_distribution_inference.py

```diff
@@ -253,6 +262,39 @@ def test_multivariate_gaussian() -> None:
     ), f"Sigma did not match: sigma = {Sigma}, sigma_hat = {Sigma_hat}"
 
 
+@pytest.mark.timeout(30)
+def test_dirichlet() -> None:
+    num_samples = 2000
+    dim = 3
+
+    alpha = np.random.gamma(shape=1, size=dim) + 0.5
```

Comment:
It would maybe be better to use `@pytest.mark.parametrize` to provide `alpha` as an argument to this function